package control_errores;

public class AdvinaError extends RuntimeException{
	
	public AdvinaError(int n) {
		super(n + " no es el n√∫mero, intentalo otra vez");
	}

}
